src/react/Filters/hooks/useOnlySemverReleases/useOnlySemverReleases.ts

```diff
@@ -1,16 +1,7 @@
-import { value } from "../../../../can";
```

Comment:
here's an example of what a refactor using the `useRouteData` hook would look like

Reply:
clean